05_stringer/joshcarp/IPAddr.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import "fmt"
+
+type IPAddr struct {
```

Comment:
Probably trying to get to clever here, [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) means that there isn't really fixed purpose for the octets.

Reply:
`[4]byte` rather than a struct is appropriate

Reply:
Yeah well my point was this was my approach before I knew that ```[4]byte``` could be used, and like in the spirit of not copying peoples code and all

Reply:
I'm not quite sure how to respond to this, i agree it's a good approach to develop your code before looking at others (i'm doing that too). It's also a normal code review process to receive feedback and improve code in response to feedback (or argue the case why you think your approach is better :) ).